src/data/styles/advances-in-complex-systems.csl

```diff
@@ -0,0 +1,128 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
For the moment copy and paste of the csl files is okay, but in the longterm it would be better to include them via a git submodule. In this way they can be updated very easily.